src/transformers/models/auto/modeling_tf_auto.py

```diff
@@ -414,12 +431,14 @@
         (XLMRobertaConfig, TFXLMRobertaForTokenClassification),
         (LongformerConfig, TFLongformerForTokenClassification),
         (RobertaConfig, TFRobertaForTokenClassification),
+        (LayoutLMConfig, TFLayoutLMForTokenClassification),
         (BertConfig, TFBertForTokenClassification),
         (MobileBertConfig, TFMobileBertForTokenClassification),
         (XLNetConfig, TFXLNetForTokenClassification),
         (ElectraConfig, TFElectraForTokenClassification),
         (FunnelConfig, TFFunnelForTokenClassification),
         (MPNetConfig, TFMPNetForTokenClassification),
+        (LayoutLMConfig, TFLayoutLMForTokenClassification),
```

Comment:
```suggestion
```
It's already defined up there.

Reply:
fixed [here](https://github.com/huggingface/transformers/pull/10636/commits/3bee70daf2a71066335d360761ce5c7bb432500a)